src/data/reports/report.ts

```diff
@@ -1,6 +1,6 @@
 export interface Reports {
-    name: string;
-    lastname: string;
-    primaryContact: string;
-    appointment: string;
+	name: string;
+	lastname: string;
+	primaryContact: string;
+	appointment: string;
```

Comment:
if is the same structure for an user you can replace your class for
`Reports {
patient: Patient,
appointment: string
}
`

Reply:
I have already made the changes